util/sdk/macros/index.mjs

```diff
@@ -354,8 +416,38 @@ const generateImports = json => {
   return imports
 }
 
-const generateInitialization = json => generateEventInitialization(json) + '\n' + generateProviderInitialization(json) + '\n' + generateDeprecatedInitialization(json)
+const generateInitialization = json => [generateEventInitialization(json), generateProviderInitialization(json), generateDeprecatedInitialization(json)].join('\n')
+
+const generateConfig = (json, pkg) => [generateHttpConfig(json, pkg)].join('\n')
+
+const generateHttpConfig = (json, pkg) => {
+  if (httpMethodsOrEmptyArray(json).length) {
+    const config =
+`Configuration.set('apiBaseUri', '${json.info["x-http-endpoint"]}')
```

Comment:
This is teetering on enough code to be in a js file instead of a string

Reply:
Absolutely agree. I think my next branch will be to remove every bit of hard coded JavaScript from our codebase, which is the first step towards enabling other language SDKs to be generated, e.g. RUST or C.

I added a `src/template/js/configuraiton` directory that works the same way as `src/templates/js/methods` and put this code in it under `http.js` (with an empty `default.js`).